<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <listeners>
    <listener class-name="Utilities.ExtentReportManager"/>
  </listeners>
  <test thread-count="5" name="Test(failed)">
    <groups>
      <run>
        <include name="Login"/>
      </run>
    </groups>
    <classes>
      <class name="LoginTestCases.TC0004_LoginTest">
        <methods>
          <include name="verifyCurrentErrorMessage" invocation-numbers="0"/>
          <include name="login"/>
          <include name="teardown"/>
          <include name="setup"/>
        </methods>
      </class> <!-- LoginTestCases.TC0004_LoginTest -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
